automatic/kubernetes-cli/legal/VERIFICATION.txt

```diff
@@ -4,12 +4,12 @@ Verification is intended to assist the Chocolatey moderators and community
 in verifying that this package's contents are trustworthy.
 
 The embedded software have been downloaded from the listed download
-location on <https://github.com/kubernetes/kubernetes/releases/latest>
+location on <https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.24.md>
 and can be verified by doing the following:
 
 1. Download the following:
-  32-bit software: <https://dl.k8s.io/v1.24.1/kubernetes-client-windows-386.tar.gz>
```

Comment:
No need to include the changes in this file, as AU will update it when it updates the package.

Reply:
I've dropped this files changes. Pleas verify.